docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -285,7 +285,9 @@ The following values are accepted for this property:
 Querying PostgreSQL
 -------------------
 
-The PostgreSQL connector provides a schema for every PostgreSQL schema.
+The PostgreSQL connector supports reading PostgreSQL system tables, e.g., ``pg_catalog`` relations. 
```

Comment:
Please rebase on master to resolve conflicts. The file was converted to markdown.